frontend/src/types/TabSections.ts

```diff
@@ -0,0 +1,13 @@
+enum TabSections {
+  Member = "members",
+  Roles = "roles",
+  Groups = "groups",
+  Identities = "identities",
+  ServiceTokens = "service-tokens"
+}
+
+const isTabSection = (value: any): value is TabSections => {
+  return Object.values(TabSections).includes(value);
+}

```

Comment:
for uniformity with the existing structure, let's do export here right away

Reply:
alright